examples/onnxruntime/training/docker/run.sh

```diff
@@ -16,13 +16,16 @@ python -m torch_ort.configure
 
 # Install dependencies
 pip install transformers datasets
-pip install coloredlogs absl-py rouge_score seqeval scipy sacrebleu nltk sklearn
+pip install coloredlogs absl-py rouge_score seqeval scipy sacrebleu nltk sklearn parameterized
 pip install fairscale deepspeed mpi4py
+# pip install optuna ray sigopt wandb # Hyper parameter search
 
 # Install apex
 # git clone https://github.com/NVIDIA/apex \
 #     && cd apex \
 #     && pip install -v --disable-pip-version-check --no-cache-dir --global-option="--cpp_ext" --global-option="--cuda_ext" ./
+# Inatall bitsandbytes
+# pip install bitsandbytes-cuda113
```

Comment:
Yes, I prefer to keep them. They are dependencies for hyper parameter search or optimized optimizers. I put them in the comment as it is not necessary to run them every night. But in case of large release of transformers, it would be nice to uncomment them to ensure nothing is broken.